app/support/notification_sender.rb

```diff
@@ -68,9 +68,9 @@ def reviewed_at
 
   class AccountNotifier
 
-    def notify_accounts(account_ids_to_notify)
+    def notify_accounts(account_ids_to_notify, facility)
```

Comment:
Looks like it's being called only with `#deliver_now`, so should be fine according to my understanding.